client/src/components/App.jsx

```diff
@@ -1,16 +1,53 @@
 import React from 'react';
+import axios from 'axios';
+// eslint-disable-next-line import/extensions
+import MenuCategoryButtons from './MenuCategoryButtons.jsx';
+// eslint-disable-next-line import/extensions
+// import MenusByHeaders from './MenusByHeaders.jsx';
+// // eslint-disable-next-line import/extensions
+// import MenuCollapse from './MenuCollapse.jsx';
 
 class App extends React.Component {
-    constructor(props) {
-      super(props);
-      
-    }
+  constructor(props) {
+    super(props);
+    this.state = {
+      restaurantData: {},
+      // currentCategory: '',
+      // menu : '',
+      // menuHeaders: [],
+    };
+  }
 
-    render() {
-        return (
-            <div>test success!</div>
-        )
-    }
+  componentDidMount() {
+    axios.get('http://localhost:3002/api/menuData')
```

Comment:
Instead of getting the full dataset here, maybe it's more sufficient to get only the first record of the 100 records.